workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -82,11 +83,26 @@ spec:
         - name: red
           model: e1000 # expose e1000 NIC to the guest
           bridge: {} # connect through a bridge
+          ports:
+           - name: http
+             port: 80
   networks:
   - name: red
     pod: {}
 ```
 
+### Ports
+
+Ports will be configured as container ports for the virt-launcher pod.
+
+> **Note:** When using the slirp interface only the configured ports will be forwarded to the virtual machine.
+
+| Options ||
+|--|--|
+| `name` | Name |
+| `port` | Port to expose (mandatory)|
+| `protocol` | Connection protocol (TCP, UDP)|
```

Comment:
Please add a column with "Format" that will have TCP, UDP for protocol; and 1-6... for port.